src/oapi/flexible_gpus.rs

```diff
@@ -53,6 +53,14 @@ impl Input {
     }
 
     pub fn fill_resource_flexible_gpus(&self, resources: &mut Resources) {
+        if self.flexible_gpus.is_empty() && self.need_default_resource {
+            resources.resources.push(Resource::FlexibleGpu(FlexibleGpu {
+                account_id: self.account_id(),
+                read_date_rfc3339: self.fetch_date.map(|date| date.to_rfc3339()),
+                region: self.region.clone(),
+                ..Default::default()
```

Comment:
default is nice, but will make the code inconsistant with bellow initialisation:
when there is fgpus, each gpu is initialize like that:
```rust
            let core_flexible_gpu = FlexibleGpu {
                osc_cost_version: Some(String::from(VERSION)),
                account_id: self.account_id(),
                read_date_rfc3339: self.fetch_date.map(|date| date.to_rfc3339()),
                region: self.region.clone(),
                resource_id: Some(flexible_gpu_id.clone()),
                price_per_hour,
                price_per_month: None,
                model_name: Some(model_name),
            };
            ```
            but if it's in the default init, it's like this:
            ```
            FlexibleGpu {
                       osc_cost_version: Some(String::from(VERSION)),
                       account_id: Some("".to_string()),
                       read_date_rfc3339: Some("".to_string()),
                       region: Some("".to_string()),
                       resource_id: Some("".to_string()),
                      price_per_hour: Some(0.0),
                      price_per_month: Some(0.0),
                      model_name: Some("".to_string()),
                      }
            ```
            don't know how much this is a problem, but do we want that ?

Reply:
@outscale-mgo It become with default.
        FlexibleGpu {
                       osc_cost_version: Some(String::from(VERSION)),
                account_id: self.account_id(),
                read_date_rfc3339: self.fetch_date.map(|date| date.to_rfc3339()),
                region: self.region.clone(),
                       resource_id: Some("".to_string()),
                      price_per_hour: Some(0.0),
                      price_per_month: Some(0.0),
                      model_name: Some("".to_string()),
                      }